include/boost/graph/r_c_shortest_paths.hpp

```diff
@@ -21,12 +23,12 @@ namespace boost {
 
 // r_c_shortest_paths_label struct
 template<class Graph, class Resource_Container>
-struct r_c_shortest_paths_label
+struct r_c_shortest_paths_label : public boost::enable_shared_from_this<r_c_shortest_paths_label<Graph, Resource_Container> >
 {
   r_c_shortest_paths_label
   ( const unsigned long n, 
     const Resource_Container& rc = Resource_Container(), 
-    const r_c_shortest_paths_label* const pl = 0, 
+    const boost::shared_ptr<r_c_shortest_paths_label<Graph, Resource_Container> > pl = nullptr, 
```

Comment:
nullptr is C++11, the C++03 tests will fail.

Reply:
This was fixed in #114.

Reply:
It's better to fix it here, if possible. It's easier to review and merge PRs one by one (and it would be easier to revert them later if some prove problematic.)

Reply:
You can.  I'm new and don't have write access so having one place to work on keeps things a tad more sane.  It fixing commit was 6808062c07be7e394223819ea703ccb2c98e84a1 .

Reply:
Thanks, @pdimov, I reapplied @anadon commit that removes usage of `nullptr` and removes spaces from end of lines.